.github/workflows/ci.yaml

```diff
@@ -23,3 +37,28 @@ jobs:
       run: make vet-check
     - name: coding styles
       run: make fmt-check
+    - name: build
+      run: make -j4 cross-compiled
+    - name: compress
+      run: |
+        set -x
+        for FILE in build/*; do
+          gzip $FILE
+        done
+    - uses: actions/upload-artifact@v2
+      with:
+        name: trento-binaries
+        path: build
+
+  upload-release-assets:
+    needs: build
+    if: github.event.release
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/download-artifact@v2
+        with:
+          name: trento-binaries
+      - uses: AButler/upload-release-assets@v2.0
```

Comment:
Is this `AButler` `AB` in capital letters in purpose? Just in case if its a typo (most probably not, but better to double check hehe)

Reply:
Yep, it's capitalized! https://github.com/AButler/upload-release-assets
I guess it would work regardless, but hey,